src/main/java/nl/dtls/fairdatapoint/service/metadata/validator/MetadataValidator.java

```diff
@@ -62,7 +65,7 @@
 
     public void validate(Model metadata, IRI uri, ResourceDefinition rd) throws MetadataServiceException {
         validateByShacl(metadata, uri);
-        if (!rd.getName().equals("Repository")) {
+        if (!rd.getUrlPrefix().equals("")) {
```

Comment:
Is there a need to keep the special handling of repository? And would there be a case for `.isBlank()` or `.isEmpty()` vs `.equals("")`?

Reply:
Yes, it would fail then when checking the parent (as parent is mandatory for all other resource definitions, the single repository is the only possible "root"). But changed to `.isEmpty()`.

`.isBlank()` would return true for whitespaces (which are different strings actually). It is expecting that resource definition URI prefix is entered somewhat correctly for use in URI. Maybe we should add extra validations and trimming...